modules/login/test/Request_Account_test_plan.md

```diff
@@ -1,13 +1,11 @@
-#Request account feature - Test Plan 
-*Stored under User Accounts module test/ directory*
+# Request Account - Test Plan 
  
 ### Request Account form: 
 1. Verify that form returns error message for each field left blank
-2. Verify that form returns appropriate error message for each field with invalid value entered (e.g.invalid email)
+2. Verify that form returns appropriate error message for each field with invalid value entered (e.g. invalid email)
 3. Verify that verification code is enforced (if and only if re-captcha has been activated by the project)
-4. Verify that new verification code loads on every refresh of the page
```

Comment:
This refers to a very very old custom captcha that we deleted because it was insecure. https://github.com/aces/Loris/pull/3645